onnxruntime/core/framework/error_code.cc

```diff
@@ -23,7 +23,7 @@ struct OrtStatus {
 _Check_return_ _Ret_notnull_ OrtStatus* ORT_API_CALL OrtApis::CreateStatus(OrtErrorCode code,
                                                                            _In_z_ const char* msg) NO_EXCEPTION {
   assert(!(code == 0 && msg != nullptr));
-  SafeInt<size_t> clen(strlen(msg));
+  SafeInt<size_t> clen(nullptr == msg ? 0 : strlen(msg));
```

Comment:
Did you run the the build in debug mode or release mode when you saw the warning? I suggest we should run all the static analyzers  in debug mode.

Reply:
How to set xcode to run analyze in a certain mode?